src/schema/rules/sidecars/pet.yaml

```diff
@@ -276,3 +276,14 @@ PETTask:
     TaskDescription: recommended
     CogAtlasID: recommended
     CogPOID: recommended
+
+DeidentificationMethod:
+  selectors:
+    - intersects([modality], ["mri", "pet"])
+  fields:
+    DeidentificationMethod:
+      level: optional
+      description_addendum: Corresponds to DICOM Tag 0012, 0063 `De-identification Method`.
+    DeidentificationMethodCodeSequence:
+      level: optional
+      description_addendum: Corresponds to DICOM Tag 0012, 0064 `De-identification Method Code Sequence`.
```

Comment:
We don't actually need this duplicated, since the `- intersects(...)` selector above ensures it will apply to MRI and PET.

```suggestion
```